src/components/Resume/Header.js

```diff
@@ -25,71 +25,95 @@ export const Header = ({
   font,
   showIcon,
 }) => (
-  <header className="resume-header" style={{ fontFamily: font }}>
-    <h1 style={{ fontFamily: font }}>{header.name}</h1>
-    <ul>
-      {showEmail
-        && (
-          <li data-testid="Email">
-            <a href={`mailto:${header.email}?subject=Interview%20Request`}>
-              { showIcon ? <img src={mailIcon} className="header-icon normal-icon" alt="Mail Icon" /> : '' }
-              { showIcon ? <img src={mailDarkIcon} className="header-icon dark-icon" alt="Mail Icon" /> : '' }
-              {header.email}
-            </a>
-          </li>
-        )}
-      {showPhone
-        && (
-          <li data-testid="Phone">
-            <a href={`tel:${header.phone}`}>
-              { showIcon ? <img src={phoneIcon} className="header-icon normal-icon" alt="Phone Icon" /> : '' }
-              { showIcon ? <img src={phoneDarkIcon} className="header-icon dark-icon" alt="Phone Icon" /> : '' }
-              {header.phone}
-            </a>
-          </li>
-        )}
-      {showGithub
-        && (
-          <li data-testid="Github">
-            <a href={header.github} target="_new">
-              { showIcon ? <img src={githubIcon} className="header-icon normal-icon" alt="Github Icon" /> : '' }
-              { showIcon ? <img src={githubDarkIcon} className="header-icon dark-icon" alt="Github Icon" /> : '' }
-              {header.github}
-            </a>
-          </li>
-        )}
-      {showLinkedIn
-        && (
-          <li data-testid="LinkedIn">
-            <a href={header.linkedin} target="_new">
-              { showIcon ? <img src={linkedinIcon} className="header-icon normal-icon" alt="LinkedIn Icon" /> : '' }
-              { showIcon ? <img src={linkedinDarkIcon} className="header-icon dark-icon" alt="LinkedIn Icon" /> : '' }
-              {header.linkedin}
-            </a>
-          </li>
-        )}
-      {showWebsite
-        && (
-          <li data-testid="Website">
-            <a href={header.website} target="_new">
-              { showIcon ? <img src={websiteIcon} className="header-icon normal-icon" alt="Website Icon" /> : '' }
-              { showIcon ? <img src={websiteDarkIcon} className="header-icon dark-icon" alt="Website Icon" /> : '' }
-              {header.website}
-            </a>
-          </li>
-        )}
-    </ul>
-    {showAddress && (
-      <ul data-testid="Address">
-        <li>{header.address}</li>
-        <li>{header.city}</li>
-        <li>{header.state}</li>
-        <li>{header.zip}</li>
-        <li>{header.country}</li>
+    <header className="resume-header" style={{ fontFamily: font }}>
+      <div>
+        <img src={header.picture} alt={`${header.name}-picture`} style={{ borderRadius: '50%', width: 'auto', objectFit: 'cover', height: '70px' }} />
```

Comment:
I am going to add some clarification to the instructions. The header image is good but, I want to include the ability to toggle it on and off. Additionally, I am going to add a helper method to migrate the old schema to the new schema.

Reply:
okay, I would add the ability to toggle it on and off. Also, the helper method to migrate the old schema I think you lost me right there.